_posts/2017-11-11-environments.md

```diff
@@ -0,0 +1,115 @@
+---
+layout: post
+title: Managing software versioning using Conda environments
+---
+
+Research projects can often take months to years to complete, however the
+precise version of software they use will often have a shorter lifetime than
+this.  These new versions of software will often include new features which
+might be of great use, but they might also introduce changes which break
+your existing work and introduce compatibility issues with other pieces
+of software.  So whilst for existing projects we might wish to freeze
+all pieces of software installed, for newer projects we instead want to
+use the most up-to-date versions, leaving us needing to install multiple
+versions of multiple different pieces of software.
+
+In this post we will try to explain how conda and Python virtual environments
```

Comment:
Or even better: make these links to the sections below. With the new AnchorJS extension all headings will get anchors so you just have to run locally to find the anchor name and put in a link, something like

      [conda](#conda-environments)